src/ascend.ts

```diff
@@ -229,36 +234,59 @@ function isInValhalla(): boolean {
 }
 
 /**
- * Hops the gash, perming no skills
+ * Hops the gash, perming no skills by default
  *
- * @param path path of choice, as a Path object--these exist as properties of Paths
- * @param playerClass Your class of choice for this ascension
- * @param lifestyle 1 for casual, 2 for softcore, 3 for hardcore. Alternately, use the Lifestyle enum
- * @param moon Your moon sign as a string, or the zone you're looking for as a string
- * @param consumable From the astral deli. Pick the container item, not the product.
- * @param pet From the astral pet store.
- * @param permOptions Options for perming during a player's stay in Valhalla
- * @param permOptions.permSkills A Map<Skill, Lifestyle> of skills you'd like to perm, ordered by priority.
- * @param permOptions.neverAbort Whether the ascension shouold abort on failure
+ * @param inputValues Configuration for the ascension
+ * @param inputValues.path path of choice, as a Path object--these exist as properties of Paths
+ * @param inputValues.playerClass Your class of choice for this ascension
+ * @param inputValues.lifestyle 1 for casual, 2 for softcore, 3 for hardcore. Alternately, use the Lifestyle enum
+ * @param inputValues.kolGender An entry from the KolGender enum: 1 for male, 2 for female (sorry that it's limited to those). Defaults to 2 or the corresponding value for defaultGenderOverride pref (which should be 'male' or 'female')
+ * @param inputValues.moon Your moon sign as a string, or the zone you're looking for as a string
+ * @param inputValues.consumable From the astral deli. Pick the container item, not the product. Defaults to astral six-pack, provide $item`none` for nothing.
+ * @param inputValues.pet From the astral pet store.
+ * @param inputValues.permOptions Options for perming during a player's stay in Valhalla
+ * @param inputValues.permOptions.permSkills A Map<Skill, Lifestyle> of skills you'd like to perm, ordered by priority.
+ * @param inputValues.permOptions.neverAbort Whether the ascension should abort on failure
  */
-export function ascend(
-  path: Path,
-  playerClass: Class,
-  lifestyle: Lifestyle,
-  moon: InputMoonSign,
-  consumable: Item | undefined = $item`astral six-pack`,
-  pet: Item | undefined = undefined,
-  permOptions?: { permSkills: Map<Skill, Lifestyle>; neverAbort: boolean }
-): void {
+export function ascend(inputValues: {
+  path: Path;
+  playerClass: Class;
+  lifestyle: Lifestyle;
+  kolGender?: KolGender;
+  moon: InputMoonSign;
+  consumable?: Item;
+  pet?: Item;
+  permOptions?: { permSkills: Map<Skill, Lifestyle>; neverAbort: boolean };
+}): void {
+  const DEFAULT_VALUES = {
+    kolGender:
```

Comment:
kol stores gender as an int; `1` for male and `2` for female. Passing this as-is into the URL should cause an error

Reply:
The enum stores them as such, so it should be fine shouldn't it?

Reply:
dang I could have sworn a minute ago these were plain strings. Yeah this is fine

Reply:
You're not crazy, they were initially, but not by the time I actually put up the PR, I think? Were you looking at it when I pushed the branch before I put up the PR maybe? Although at that point I believe I did have it converting to 1 or 2 in the url, just further below.